plugins/inputs/statsd/statsd.go

```diff
@@ -348,7 +348,7 @@ func (s *Statsd) parseStatsdLine(line string) error {
 				return errors.New("Error Parsing statsd line")
 			}
 			m.floatvalue = v
-		case "c", "s":
+		case "c", "s", "m", "mr":
```

Comment:
`m` - a `meter` metric; increments by 1 (eg.: on each request)

```
1|478
bifrost.application.byStatusCode.200:1|m
```

https://github.com/chef/chef-server/blob/7c1df38aee7439674972a49bc157443b5c5c9cbf/src/oc_bifrost/doc/metrics.md#end-of-request

This one will be integer for sure.